.github/workflows/version-bump.yml

```
@@ -116,29 +116,33 @@ jobs:
+          
+          # Create PR with auto-merge
+          gh pr create \
+            --title "chore: bump $BUMP_TYPE version to $NEW_VERSION" \
+            --body "Automated version bump from merged PR #${{ github.event.pull_request.number }}" \
+            --base main \
+            --head "$BRANCH_NAME"
+          
+          # Enable auto-merge if available
+          PR_NUMBER=$(gh pr view "$BRANCH_NAME" --json number -q .number)
+          gh pr merge "$PR_NUMBER" --auto --squash || echo "Auto-merge not available, manual merge required"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)